### ISSUE TITLE
Adding a ML Theory Branch

### ISSUE THREAD

**Issue by EtashGuha** [emeryberger/CSrankings#5677]
Hey guys, love the tools!
As a ML Theory guy myself, it would be great to see an AISTATS and COLT section for ML Theory. Thanks!

**Comment by trucndt** [comment]
I agree with including conferences like AISTATS. A previous issue also suggested the same https://github.com/emeryberger/CSrankings/issues/5428
